packages/services/src/providers/__tests__/multi-chain.provider.test.ts

```diff
@@ -160,4 +163,141 @@ describe("MultiChainProvider", () => {
       }
     });
   });
+
+  describe("Configurable price providers", () => {
```

Comment:
just a callout that with this feature, we can easily switch between price providers; all we need to do is update an env var and restart the server